ext/nnstreamer/tensor_filter/meson.build

```diff
@@ -859,6 +859,27 @@ if executorch_support_is_available
   )
 endif
 
+if executorch_llama_support_is_available
+  filter_sub_executorch_sources = ['tensor_filter_executorch_llama.cc']
+  nnstreamer_filter_executorch_llama_deps = [executorch_llama_support_deps, glib_dep, nnstreamer_single_dep]
+  nnstreamer_filter_executorch_llama_deps += declare_dependency(compile_args: ['-Wno-sign-compare'])
+  nnstreamer_filter_executorch_llama_deps += declare_dependency(compile_args: ['-Wno-deprecated-declarations'])
```

Comment:
suggestion:
```
executorch_llama_compile_args = cxx.get_supported_arguments([
  '-Wno-sign-compare', '-Wno-deprecated-declarations'
])

nnstreamer_filter_executorch_llama_deps += declare_dependency(compile_args: executorch_llama_compile_args)
```

Reply:
That's clean. I updated the code as you suggested.